Common/include/option_structure.hpp

```diff
@@ -300,7 +300,7 @@ enum ENUM_INTERPOLATOR {
   NEAREST_NEIGHBOR 	= 0,   	/*!< \brief Nearest Neigbhor interpolation */
   ISOPARAMETRIC 	= 1,	/*!< \brief Isoparametric interpolation */
   CONSISTCONSERVE 	= 2,	/*!< \brief Consistent & Conservative interpolation (S.A. Brown 1997). Utilizes Isoparametric interpolation. */
-  SLIDING_MESH		= 3, 	/*!< \brief Sliding Mesh Approach E. Rinaldi 2015 */
+  WEIGHTED_AVERAGE  = 3, 	/*!< \brief Sliding Mesh Approach E. Rinaldi 2015 */
 };
```

Comment:
I see you opted for WEIGHTED_AVERAGE eventually. I would then also change the corresponding class name (and maybe the comment for the description?).